rebar.config

```diff
@@ -38,6 +38,11 @@
                              {vm_args, "./config/dev1/vm.args"}]}
                     ]
             },
+            {fastmining, [{relx, [{dev_mode, false},
```

Comment:
I understand this profile will be deleted once the production release has mining speed configurable, right?
(no need to amend PR for this - just checking we have same understanding)

Reply:
Yes, exactly